generate-release/src/github_client.rs

```diff
@@ -198,19 +198,6 @@ impl GithubClient {
 
             prs.append(&mut prs_in_page);
             page += 1;
-            if let Some(pr) = prs.last() {
-                if let Some(datetime_utc) = datetime_utc {
-                    if let Some(closed_at) = pr.closed_at {
-                        if closed_at < datetime_utc {
```

Comment:
This data seems to be unreliable, and early filtering here doesn't actually do us any good.

Reply:
I agree that this filtering seems pointless